projects/1.19/assets/dungeons-mobs/dungeons_mobs/lang/zh_cn.json

```
@@ -0,0 +1,524 @@
+  "item.dungeons_mobs.royal_guard_shield.orange": "橙色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.white": "白色皇家守卫盾牌",
+
+  "itemGroup.dungeonsMobs": "地下城：生物",
+  "itemGroup.dungeonsMobsItems": "地下城：生物装备",
+
+  "mobEnchantment.dungeons_mobs.burning": "灼烧",
+  "mobEnchantment.dungeons_mobs.chilling": "冻结",
+  "mobEnchantment.dungeons_mobs.deflect": "偏转",
+  "mobEnchantment.dungeons_mobs.double_damage": "双倍伤害",
+  "mobEnchantment.dungeons_mobs.echo": "回击",
+  "mobEnchantment.dungeons_mobs.fire_trail": "焰尾",
+  "mobEnchantment.dungeons_mobs.gravity_pulse": "引力脉冲",
+  "mobEnchantment.dungeons_mobs.levitation_shot": "悬浮射击",
+  "mobEnchantment.dungeons_mobs.multishot": "多重",
+  "mobEnchantment.dungeons_mobs.protection": "保护",
+  "mobEnchantment.dungeons_mobs.quick": "快速",
+  "mobEnchantment.dungeons_mobs.tempo_theft": "速度窃取",
+  "mobEnchantment.dungeons_mobs.thorns": "荆棘",
+
+  "mobenchant.enchantwithmob.name.dungeons_mobs.regeneration": "再生",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.rush": "冲刺",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.radiance": "治愈光辉",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.gravity_pulse":  "引力脉冲",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.burning": "灼烧",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.chilling":  "冻结",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.heals_allies": "治疗友方",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.deflect":  "偏转",
+  "mobenchant.enchantwithmob.name.dungeons_mobs.echo": "回击",
+  
+  "subtitles.entity.wraith.idle": "幽灵：哀叫",
+  "subtitles.entity.wraith.hurt": "幽灵：受伤",
+  "subtitles.entity.wraith.death": "幽灵：死亡",
+  "subtitles.entity.wraith.attack": "幽灵：攻击",
+  "subtitles.entity.wraith.fire": "幽灵：召唤火",
+  "subtitles.entity.wraith.fly": "幽灵：滑翔",
+  "subtitles.entity.wraith.teleport": "幽灵：传送",
+  
+  "subtitles.entity.enchanter.idle": "附魔师：抱怨",
+  "subtitles.entity.enchanter.hurt": "附魔师：受伤",
+  "subtitles.entity.enchanter.death": "附魔师：死亡",
+  "subtitles.entity.enchanter.pre_attack": "附魔师：举起书",
+  "subtitles.entity.enchanter.attack": "附魔师：攻击",
+  "subtitles.entity.enchanter.spell": "附魔师：准备附魔",
+  "subtitles.entity.enchanter.beam": "附魔师：附魔",
+  
+  "subtitles.entity.geomancer.idle": "地卜师：怪笑",
+  "subtitles.entity.geomancer.hurt": "地卜师：受伤",
+  "subtitles.entity.geomancer.death": "地卜师：死亡",
+  "subtitles.entity.geomancer.pre_attack": "地卜师：准备施法",
+  "subtitles.entity.geomancer.attack": "地卜师：召唤石柱",
+  "subtitles.entity.geomancer.wall_spawn": "石柱墙：生成",
+  "subtitles.entity.geomancer.wall_despawn": "石柱墙：消失",
+  "subtitles.entity.geomancer.bomb_spawn": "爆炸石柱墙：生成",  
+  
+  "subtitles.entity.squall_golem.idle": "呼啸傀儡：嘎吱作响",
+  "subtitles.entity.squall_golem.hurt": "呼啸傀儡：受伤",
+  "subtitles.entity.squall_golem.death": "呼啸傀儡：死亡",
+  "subtitles.entity.squall_golem.attack": "呼啸傀儡：攻击",
+  "subtitles.entity.squall_golem.activate": "呼啸傀儡：激活",
+  "subtitles.entity.squall_golem.deactivate": "呼啸傀儡：关闭",
+  "subtitles.entity.squall_golem.step": "呼啸傀儡：行走",

```

```suggestion
"subtitles.entity.squall_golem.step": "呼啸傀儡：脚步声",
```
参考[wiki的字幕翻译](https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95)